_sass/plain.scss

```diff
@@ -104,7 +107,9 @@ h2 {
 
   a {
     color: vars.$link-color;
-    text-decoration: none;
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+    text-underline-offset: 0.2em;
     transition: border-bottom 0.4s;
```

Comment:
I believe this `border-bottom` transition was adding an animation on hover which is no longer needed. Please remove.